Reference/google/protobuf/any.pb.swift

```diff
@@ -108,18 +108,14 @@ import Foundation
 ///         "@type": "type.googleapis.com/google.protobuf.Duration",
 ///         "value": "1.212s"
 ///       }
-struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message {
+struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message, ProtoNameProviding {
   public var swiftClassName: String {return "Google_Protobuf_Any"}
   public var protoMessageName: String {return "Any"}
   public var protoPackageName: String {return "google.protobuf"}
-  public var jsonFieldNames: [String: Int] {return [
-    "typeUrl": 1,
-    "value": 2,
-  ]}
-  public var protoFieldNames: [String: Int] {return [
-    "type_url": 1,
-    "value": 2,
-  ]}
+  public static let _protobuf_fieldNames: FieldNameMap = [
+    1: .unique(proto: "type_url", json: "typeUrl", swift: "typeURL"),
+    2: .same(proto: "value", swift: "value"),
```

Comment:
I know you've been agitating to drop the swift names, but if all three are usually the same, there may not be much gain by dropping them.  (And I think there are small but positive advantages for mirror and debug description to use the  swift field names instead of the proto names.)